examples/chart/teleport-cluster/templates/proxy/service.yaml

```diff
@@ -0,0 +1,66 @@
+{{- $proxy := mustMergeOverwrite .Values .Values.proxy -}}
+{{- $backendProtocol := ternary "ssl" "tcp" (hasKey $proxy.annotations.service "service.beta.kubernetes.io/aws-load-balancer-ssl-cert") -}}
```

Comment:
I still kinda don't like that we have this "magic" behaviour based solely on the AWS ACM annotation, but we can always tidy this up a bit or make it more explicit in future (maybe once we start being able to support Ingress properly)

Reply:
At least now we'll honor the labels you pass instead of forcing ours. This should fix https://github.com/gravitational/teleport/issues/16220